### PR TITLE
CVE-2020-13776 (Updated)

### PR DESCRIPTION
**Pull request by axg3073**
Finish large majority to all of filling in answers to the questions and other criteria. Unsure of the success of CI running correctly at all when pushed on GitHub.

### CODE REVIEW

File: cves/CVE-2020-13776.yml [hunk 1]
```diff
@@ -70,7 +72,7 @@ bugs_instructions: |
 
   For systemd, this is typically their GitHub issues, but could also include 
   bugs from other databases. Put a URL instead of a single number.
-bugs: []
+bugs: [https://github.com/systemd/systemd/issues/15985]
```
Comment (andymeneely) [inline cves/CVE-2020-13776.yml:75]: They also link to another bug report: https://github.com/systemd/systemd/issues/6237

File: cves/CVE-2020-13776.yml [hunk 2]
```diff
@@ -203,20 +223,28 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-
-  name:
-  note:
+        
+    Name should be all lowercase, NOT a specific file.
+  name: ['user-util.c', 'parse-util.c', 'parse-util.h']
```
Comment (andymeneely) [inline cves/CVE-2020-13776.yml:228]: This is currently causing the failure. I'm thinking a subsystem as larger than a single file. From the filenames, perhaps "util", but then looking at what the bug is it sounds like "user" might be a good name for the subsystem. Try to pick a name that someone else might pick if theirs were in a similar place.

File: cves/CVE-2020-13776.yml [hunk 3]
```diff
@@ -278,15 +305,20 @@ discussion:
...
 
     These answers should be boolean.
-      discussed_as_security: true or false
-      any_discussion: true or false
+      disputed_as_security: true
+      any_disagreement: false
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
+      discussed_as_security: true
```
Comment (andymeneely) [inline cves/CVE-2020-13776.yml:313]: Looks like this is indented farther in, which YML puts as part of the long string before it. Unindent to the level of "question" on line 289 and it should pass.

Comment (axg3073) [reply cves/CVE-2020-13776.yml]: > You mean unindent 311 and after to same level of "question 289" Line 307 also seems to have answers within same indent level. I believe these lines were all indented to that level that is currently

Nvm, the indention at 313 is what you meant. Seems to pass now

File: cves/CVE-2020-13776.yml [hunk 4]
```diff
@@ -55,7 +48,16 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Systemd used to mishandle how numerical usernames get handled.
```
Comment (WilsonJays) [inline cves/CVE-2020-13776.yml:52]: "Systemd used to mishandle how numerical usernames were handled." reads better IMO. It also keeps the tense the same.

File: cves/CVE-2020-13776.yml [hunk 5]
```diff
@@ -203,20 +224,28 @@ subsystem:
...
 
     Use this to specify any commits you think are notable in some way, and
     explain why in the note.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: '156a5fd297b61bce31630d7a52c15614bf784843'
+    note: 'Orignaly at first to be belived as a temp fixed, but actually ended up becoming the actual fix'
```
Comment (WilsonJays) [inline cves/CVE-2020-13776.yml:242]: That's funny

File: cves/CVE-2020-13776.yml [hunk 6]
```diff
@@ -226,11 +255,10 @@ i18n:
...
 
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+    Answer should be boolean. Write a note about how you came to the conclusions
+    you did.
+  answer: false
+  note: 'This should be at all any il8n issue, just a parse issue'
```
Comment (WilsonJays) [inline cves/CVE-2020-13776.yml:261]: I do not fully understand what this is trying to say. I would recommend rewording this line to be a bit more clear of what you are trying to convey.

File: cves/CVE-2020-13776.yml [hunk 7]
```diff
@@ -278,15 +306,20 @@ discussion:
...
+      disputed_as_security: true
+      any_disagreement: false
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
+  discussed_as_security: true
+  any_discussion: true
+  note: |
+    https://github.com/systemd/systemd/issues/6237
+    CVE-2017-1000082, the 'parrent' of CVE-2020-13776 as it was a no fully fixed issue for the CVE
```
Comment (WilsonJays) [inline cves/CVE-2020-13776.yml:318]: Parent is misspelled

**Comment by WilsonJays** [comment]
Upvote: 3
Interesting overall. There are just some wonky wordings.

**Pull request merged**